docs/configuration-reference.md

```diff
@@ -111,6 +111,18 @@ key | value | description
 :--|:--|:--
 field | string | The name of the field to drop.
 
+### rename_field
+
+The `rename_field` processor will rename the specified field in all events, if it exists, before sending them to Honeycomb. You can use this to standardize field names across data sources, for example.
```

Comment:
Consider mentioning that the processor will overwrite an existing target field with the "new" name.